### PR TITLE
Apply react routing to the entire App

### PR DESCRIPTION
**Pull request by ahmedadel56**
In this branch, I add another Two pages and applied to react routing on it

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json [hunk 1]
```diff
@@ -1,15 +1,16 @@
 {
-  "homepage": "https://ahmedadel56.github.io/Calculator",
   "name": "react-calculator",
   "version": "0.1.0",
   "private": true,
+  "homepage": "https://ahmedadel56.github.io/Calculator",
```
Comment (mikenath223) [inline package.json:5]: - [x] Kindly deploy the actual project online and include the correct livelink in the readme.

Comment (ahmedadel56) [reply package.json:5]: Thank You a lot for that note, I fixed that issue kindly tell me your opinion

**Review by EricMbouwe — APPROVED** [review]
### Hi @ahmedadel56, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**